### PR TITLE
Do not ignore `[[nodiscard]]` functions in example code

### PR DESCRIPTION
**Pull request by ChrisThrasher**
## Description

I wanted to make a PR out of this old commit Vittorio wrote since it's still relevant today.

### CODE REVIEW

File: include/SFML/Graphics/Texture.hpp [hunk 1]
```diff
@@ -120,8 +120,7 @@ class SFML_GRAPHICS_API Texture : GlResource
     /// This function is a shortcut for the following code:
     /// \code
     /// sf::Image image;
-    /// image.loadFromFile(filename);
-    /// texture.loadFromImage(image, area);
+    /// image.loadFromFile(filename) && texture.loadFromImage(image, area)
```
Comment (kimci86) [inline include/SFML/Graphics/Texture.hpp:123]: Something is missing, either an if statement or a semicolon. I suggest an if statement.
Same goes for other changes in this file.

**Review by kimci86 — CHANGES_REQUESTED** [review]

File: include/SFML/Graphics/Texture.hpp (hunk 1 above)
Comment (eXpl0it3r) [reply include/SFML/Graphics/Texture.hpp:123]: Yep, just turning them into a boolean expression is more confusing 😄

Comment (ChrisThrasher) [reply include/SFML/Graphics/Texture.hpp:123]: https://github.com/SFML/SFML/blob/157feec05dbf888bfbfaabf8c4ae0cb366a7f9e8/src/SFML/Graphics/Texture.cpp#L264-L265

That confused me too. It's just a snippet from the implementation of `sf::Texture::loadFromFile`.

Comment (eXpl0it3r) [reply include/SFML/Graphics/Texture.hpp:123]: Ah, I guess adding the `return` might also be okay

Comment (eXpl0it3r) [reply include/SFML/Graphics/Texture.hpp:123]: Are you gonna add this?

Comment (ChrisThrasher) [reply include/SFML/Graphics/Texture.hpp:123]: Yeah I suppose I'll eventually do that. This PR is just not a priority for me.

**Comment by ChrisThrasher** [comment]
I rebased and fixed up those comments to read like real C++ code.

**Review by eXpl0it3r — APPROVED** [review]

**Pull request merged**